predict.py

```diff
@@ -0,0 +1,157 @@
+                    transforms.Resize((256, 256)),
+                    transforms.ToTensor()])
+            elif key == 'celebs_seg_to_face':
+                self.transforms[key] = transforms.Compose([
+                    transforms.Resize((256, 256)),
+                    augmentations.ToOneHot(n_classes=19),
+                    transforms.ToTensor()])
+            elif key == 'celebs_super_resolution':
+                self.transforms[key] = transforms.Compose([
+                    transforms.Resize((256, 256)),
+                    augmentations.BilinearResize(factors=[16]),
+                    transforms.Resize((256, 256)),
+                    transforms.ToTensor(),
+                    transforms.Normalize([0.5, 0.5, 0.5], [0.5, 0.5, 0.5])])
+
+
+    @cog.input("model", type=str, options=["celebs_seg_to_face", "celebs_sketch_to_face", "ffhq_encode",
+                                                "ffhq_frontalize", "celebs_super_resolution", "toonify"],
+               help="choose model type")
+    @cog.input("image", type=Path, help="input facial image")
+    def predict(self, model, image):
+        opts = self.opts[model]
+        opts = Namespace(**opts)
+        pprint.pprint(opts)
+
+
+        net = pSp(opts)
+        net.eval()
+        net.cuda()
+        print('Model successfully loaded!')
+
+        original_image = Image.open(str(image))
+        if opts.label_nc == 0:
+            original_image = original_image.convert("RGB")
+        else:
+            original_image = original_image.convert("L")
+        original_image.resize((256, 256))
+
+        # Align Image
+        if model not in ["celebs_sketch_to_face", "celebs_seg_to_face"]:
+            input_image = run_alignment(str(image))
+        else:
+            input_image = original_image
+
+        img_transforms = self.transforms[model]
+        transformed_image = img_transforms(input_image)
+
+        if model in ["celebs_sketch_to_face", "celebs_seg_to_face"]:
+            latent_mask = [8, 9, 10, 11, 12, 13, 14, 15, 16, 17]
+        else:
+            latent_mask = None
+
+        with torch.no_grad():
+            result_image = run_on_batch(transformed_image.unsqueeze(0), net, latent_mask)[0]
+        input_vis_image = log_input_image(transformed_image, opts)
+        output_image = tensor2im(result_image)
+
+        if model == "celebs_super_resolution":
+            res = np.concatenate([np.array(input_vis_image.resize((256, 256))),
+                                  np.array(output_image.resize((256, 256)))], axis=1)
+        else:
+            res = np.array(output_image.resize((256, 256)))
+
+        out_path = Path(tempfile.mkdtemp()) / "out.png"
+        Image.fromarray(np.array(res)).save(str(out_path))
+        return out_path
+
+
+def run_alignment(image_path):
+    import dlib
+    from scripts.align_all_parallel import align_face
+    predictor = dlib.shape_predictor("shape_predictor_68_face_landmarks.dat")
```

Comment:
I think we should set the predictor in the `setup()` so it won't need to be loaded for every `predict()` call. 
And the imports I think could also be moved to the top of the file just to save the time needed for the import.